pages/dashboard/dashboard-page.js

```diff
@@ -409,19 +438,29 @@ exports.DashboardPage = class DashboardPage extends BasePage {
     await this.addProjectButton.click();
   }
 
-  async setProjectName(newProjectName) {
-    await this.projectNameInput.pressSequentially(newProjectName);
+  async setProjectName(projectName) {
+    await this.projectNameInput.pressSequentially(projectName);
     await this.page.keyboard.press('Enter');
   }
 
-  async isProjectTitleDisplayed(newProjectName) {
-    await expect(this.projectNameTitle.first()).toHaveText(newProjectName);
+  async isProjectTitleDisplayed(projectName) {
+    await expect(this.projectNameTitle.first()).toHaveText(projectName);
+  }
+
+  async isProjectTitleNotVisible(projectName) {
+    const deletedProject = this.projectNameTitle.getByText(projectName, {
+      exact: true,
+    });
+
+    await expect(
+      deletedProject,
+      'Project title should not be visible',
+    ).not.toBeVisible();
   }
 
-  async createProject(name) {
+  async createProject(projectName) {
     await this.clickAddProjectButton();
-    await this.setProjectName(name);
-    await this.isProjectTitleDisplayed(name);
+    await this.setProjectName(projectName);
   }
 
   async renameProjectViaRightClick(newProjectName) {
```

Comment:
I noticed you've changed "newProjectName" to "projectName" but missing this. Could you update it please?